distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -59,6 +59,9 @@ ALERT;RFXCOM Binding: Lighting4 default command ids are deprecated and will be r
 ALERT;Sager Weathercaster Binding: The type of the channels 'pressure', 'temperature' and 'wind-angle' was updated. You may need to remove and create again your things in case your things were previously created using UI and you want to use one of these channels.
 ALERT;Sony Projector Binding: The type of the channel 'power' was updated. You may need to remove and create again your things in case your things were previously created using UI and you want to use this channel.
 
+[3.3.0]
+ALERT;TapoControl: L510_Series and L530_Series Things were renamed to L510 and L530 because of manufacturer changed naming with new HW-rev. You may need to remove and create again this things.
```

Comment:
```suggestion
ALERT;TapoControl: L510_Series and L530_Series Things were renamed to L510 and L530 because of manufacturer changed naming with new HW-rev. You may need to remove and create again these things.
```

Reply:
Hi,
pull request [#11967](https://github.com/openhab/openhab-addons/pull/11967) is already merged.
I think it's the better cleaner way to this new binding. So we do not have a lot of things when adding them manually. This will be confusing if there will be more than one thing for the same type of device.